bin.js

```diff
@@ -18,9 +18,9 @@ function rulesToString(editorConfigRules) {
 function getEslintRules(filePath) {
   let eslintRules;
   if (filePath !== undefined) {
-    eslintRules = transformer.getEslintRules(path.resolve(filePath));
+    eslintRules = transformer.getEslintRules(path.resolve(process.cwd(), filePath));
```

Comment:
I'm not sure about that, but in any case, I don't need this package anymore, so I don't have any interest in this PR being merged anymore anyway.